app/styles/_global.scss

```diff
@@ -3,3 +3,8 @@
 #toast-container > div { // stylelint-disable-line selector-max-id
     opacity: 1;
 }
+
+.copy-btn { // stylelint-disable-line selector-class-pattern
```

Comment:
It looks like `.copy-btn` isn't used, do we need this?

Reply:
Yes, the `{{copy-button}}` component uses this. I put it in the global because we use the component in both the file detail route and file browser component.

Reply:
Ah, okay. Fair enough